Justfile

```
@@ -0,0 +1,29 @@
+test:
+    cargo nextest run
```

If i didn't miss anything, that would run only unit-tests.
```suggestion
unit-test:
    cargo nextest run
```